style.css

```diff
@@ -341,6 +342,105 @@ body {
   line-height: 42.667px;
 }
 
+.contact-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 100%;
+}
+
+.contact-data {
+  display: flex;
+  flex-direction: column;
+  padding: 75px 24px 120px 24px;
+  border-bottom: 1px solid #dbd8d7;
+  margin-bottom: 15px;
+  height: 100%;
+}
+
+.contact-section form {
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+  height: 100%;
+}
+
+.contact-input {
+  display: flex;
+  width: 90%;
+  padding: 15px 16px 15px 16px;
+  color: #1c1a19;
+  border: none;
+  border-bottom: 1px solid #dbd8d7;
+}
+
+.contact-data h2 {
+  display: flex;
+  color: #172b4d;
+  font-family: "Inter", sans-serif;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  letter-spacing: 0;
+  text-align: center;
+  margin-bottom: 58px;
+}
+
+.contact-input::placeholder {
+  color: #979493;
+}
+
+.textarea-container {
+  display: flex;
+  background-color: #fbf8f7;
+  height: 15vh;
+  position: relative;
+}
+
+.text-area {
+  background-color: #fbf8f7;
+  color: #3c3a39;
+  font-family: "Inter", sans-serif;
+  font-size: 15px;
+  font-style: italic;
+  font-weight: 400;
+  line-height: 24px;
+  width: 100%;
+  resize: none;
+  border: none;
+}
+
+.textarea-container svg {
+  position: absolute;
+  bottom: 0;
+  right: 0;
+}
+
+.btn-contact {
+  align-self: center;
+  width: 38%;
+}
+
+.bottom-nav {
+  display: flex;
+  padding-bottom: 16px;
+  justify-content: center;
+}
+
+.bottom-nav .social-media-icons {
+  gap: 0;
+  align-items: center;
+}
+
+.bottom-nav .social-media-icons svg {
+  padding: 8px;
+  color: #505f79;
+}
+
+.end-vector-container {
+  padding: 0 121px 0 120px;
+}
+
```

Comment:
Please fix the font style in inputs in order to match the one from a template that is provided below screenshot of yours.
Also double check the padding below a button get in touch in order to match the one from the template.
![Screenshot from 2023-07-12 15-58-18](https://github.com/memicelis/my-portfolio/assets/118063058/2c6a1bbe-655a-4649-a172-bc3fad352d8a)
Figma template:
![Screenshot from 2023-07-12 15-58-25](https://github.com/memicelis/my-portfolio/assets/118063058/b297cd89-2e56-459a-9beb-26abd0e3f236)